analyzer/runtime/extract.go

```diff
@@ -241,10 +241,10 @@ func registerTokenDecrease(tokenChanges map[TokenChangeKey]*big.Int, contractAdd
 	change.Sub(change, amount)
 }
 
-func ExtractRound(blockHeader block.Header, txrs []*sdkClient.TransactionWithResults, rawEvents []*sdkTypes.Event, logger *log.Logger) (*BlockData, error) {
+func ExtractRound(blockHeader nodeapi.RuntimeBlockHeader, txrs []*nodeapi.RuntimeTransactionWithResults, rawEvents []*nodeapi.RuntimeEvent, logger *log.Logger) (*BlockData, error) {
 	blockData := BlockData{
 		Header:              blockHeader,
-		Hash:                blockHeader.EncodedHash().String(),
+		Hash:                hash.NewFrom(blockHeader).String(),
```

Comment:
Not cool. Because the header is now encoded in a slightly different struct, the hash changes.
This breaks in commit 6cab131 (`storage/runtime: Support fetching data from Cobalt. No ParatimeApiLite yet. No support for very early blocks.`)

Reply:
Fixed for post-Cobalt. There was a separate bug :/ where we parsed a header as Cobalt-style if modern-style parsing _succeeded_, rather than if it failed.
Will fix for Cobalt tomorrow.

Reply:
Fixed. Header hash is now computed before the header is converted to internal representation.